tests/integration/conftest.py

```diff
@@ -1571,7 +1634,7 @@ def qtest_env_vars():
     """Retrieve a dictionary of required environment variables for running integration tests.
 
     Returns:
-        dict(str): A dictionary of environment variables. (Case sensitive)
+        {str: obj}: A dictionary of environment variables. (Case sensitive)
```

Comment:
The JetBrains IDE does not recognize "dict(*)" as a valid docstring data type. Fixed.